src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -358,7 +358,7 @@ impl ScalarFunc {
         let arg0: Cow<'a, MyDuration> = try_opt!(self.children[0].eval_duration(ctx, row));
```

Comment:
we can remove Cow for MyDuration as well! (Another PR for this change is fine)

Reply:
Thx! I will fix it in this PR.